content/p/vulnerability-disclosure-policy.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Vulnerability disclosure policy"
+type: p
+---
+
+Do you think you've found a security flaw in AlmaLinux OS or one of our related projects? Read below for how to responsibly report it!
+
+## Reporting a Vulnerability
+
+> - [Our security.txt file](/security.txt).
+
+Taking the time to report a security vulnerability to us is greatly appreciated, and we will use ever resource at our disposal to respect your time during the reporting process. When reporting an issue, please provide **as much information** as possible, but at least:
+
+- The project and version (even better if you can identify the specific commit) where you identified the vulnerability
+- A detailed description of the steps to reproduce 
+- If appropriate, please include a proof of concept (plaintext only; no binaries)
+- Please also include your recommended remediation(s), if any, or any other concerns.
+
+> #### Do Not Send:
+> - Sensitive or personal information.
+> - Public disclosures of vulnerabilities.
+
+Our maintainers will attempt to respond to and confirm your report within 2-3 days, but if you believe your report to be *critical* to user safety and security, please note as such in the subject. We are fortunate enough to have hundreds of thousands of systems relying on the expertise of the AlmaLinux OS Team, and we take security very seriously.
+
+## Example Report
+
+```text
+- **Title**: Failed to start OSBuild Composer
```

Comment:
Is this a placeholder report? It doesn't look security related.

Reply:
I just took one from bugs.almalinux.org, I can do one with [CVE-2024-1086](https://nvd.nist.gov/vuln/detail/CVE-2024-1086) if you thing it's appropriate.

Reply:
I'd say that most of the time, security advisories related to the 99% of the packages in AlmaLinux OS will be reported upstream. Maybe we can think of a good example to put here, like, dunno something related to a vulnerability in one of our github workflows, or something specific to one of our packages or our images. More ideas are welcome.